packages/api/src/routes/endpoints/control/controlLog.js

```diff
@@ -0,0 +1,42 @@
+/*
+  Copyright 2020-2024 Lowdefy, Inc
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import { type } from '@lowdefy/helpers';
+
+async function controlLog(context, routineContext, { control }) {
+  const { logger, evaluateOperators } = context;
+  const { items } = routineContext;
+
+  logger.debug({ event: 'debug_control_log' });
+  const log = evaluateOperators({ input: control[':log'], items, location: 'TODO:' });
+  const logLevel = control[':level'] ?? 'info';
```

Comment:
Should we evaluate operators for the log level? I'm not sure?

Reply:
I don't see why not

Reply:
Updated in a355bb3cddb1ea91c8c4b6fd8a45f2af41c4dff7

Reply:
Please add test case that operators are evaluated for log level

Reply:
Added in a355bb3cddb1ea91c8c4b6fd8a45f2af41c4dff7

Reply:
thanks